src/testing/e2e/statefulset.go

```diff
@@ -0,0 +1,159 @@
+package e2e
```

Comment:
should be moved to `controllers/dynakube/dynakube_controller_test.go` as that's where it's used